src/app/components/JumpTo/index.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import JumpTo from '.';
+import { StoryArgs } from '../../models/types/storybook';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+interface Props {
+  subheadlines?: { heading: string }[];
+}
+
+const Component = ({ subheadlines = [] }: Props) => {
+  return <JumpTo jumpToHeadings={subheadlines} />;
```

Comment:
**nitpick**: Why not use the same name as the prop to the component?

```suggestion
const Component = ({ jumpToHeadings = [] }: Props) => {
  return <JumpTo jumpToHeadings={jumpToHeadings} />;
```

Reply:
Doneee